config/defaults.go

```diff
@@ -18,7 +18,8 @@ var DefaultNodeConfig = NodeConfig{
 		ListenAddress: DefaultListenAddress,
 		Seeds:         "",
 	},
-	Aggregator: false,
+	Aggregator:     false,
+	LazyAggregator: false,
```

Comment:
for now this is ok, but later when we anticipate more types of sequencers, we can probably do `--aggregator=normal| lazy | ...` etc.